src/test/java/org/jcodec/codecs/h264/H264UtilsTest.java

```diff
@@ -156,4 +156,26 @@ public void testMvList2D() {
         Assert.assertEquals(0, pair.mv1Y(1, 1));
         Assert.assertEquals(-1, pair.mv1R(1, 1));
     }
+
+    @Test
+    public void testGotoNALUnit() {
```

Comment:
Could you add tests for more complex sequences, like for ex. 00 00 03 00 01 00 00 00 03 00 00 65 00 00 01 00 00 00 01 65 00 00 01 42 00 00 03 00 49. In my opinion this sequence should give 3 NAL units:
00 00 01  <empty>
00 00 00 01  [65]
00 00 01 [42 00 00 03 00 49]

Reply:
@chugai

Reply:
why 3? we will have 4.
The first NAL unit in your case will be [00 00 03 00 01 00 00 00 03 00 00 65]
because gotoNALUnit() does not skip to the next unit. We have skipToNALUnit() for that.

Reply:
@svitvitskiy

Reply:
yes, 4